components/tcf/secondLayer/src/components/IconClose.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+export default function IconClose() {
+  return (
+    <svg viewBox="0 0 24 24">
+      <path
+        id="a"
```

Comment:
This `id="a"` is not neccesary.

Also, maybe we need to reuse the same IconClose we saw in the previous firstLayer.

Reply:
Saco el id, pero de momento dejo los dos componentes así, dado que seguro que van a cambiar las especificaciones de UX/UI